src/lib/eventHandler/pasteHandler.ts

```diff
@@ -0,0 +1,129 @@
+import { decodeResponse, isImageResponse } from '../http'
+
+const handlePasteText = (
+  doc: CodeMirror.Doc,
+  change: CodeMirror.EditorChange,
+  pastedText: string
+) => {
+  doc.replaceRange(pastedText, change.from, {
+    line: change.from.line,
+    ch: change.from.ch + pastedText.length,
+  })
+}
+
+const handlePasteUrl = async (doc: CodeMirror.Doc, change: CodeMirror.EditorChange, pastedText: string) => {
+  const taggedUrl = `<${pastedText}>`
+  const urlToFetch = pastedText
+
+  doc.replaceRange(taggedUrl, change.from, {
+    line: change.from.line,
+    ch: change.from.ch + pastedText.length,
+  })
+
+  const response = await fetch(urlToFetch, { method: 'get' })
```

Comment:
@ZeroX-DG 
Thank you for your advice. That's right.
( I overlooked because I allowed `cors` in my local development environment ...)

To get this feature working on the web version in production environment I think:

(1) CORS permission setting on the Web server side (Access-Control-Allow-Origin *)
or
(2) Communicate via proxy server that CORS is allowed

(1) is not good for security, I think.
(2) is about the server configuration, so it's a little too complicated to solve here.

How about releasing this feature once as a feature for desktop and mobile apps?
If it's OK, I'll fix this feature so that it cannot be used from the Web version.

Reply:
@zeroclock sounds good to me. In the future we could do something like this:
https://github.com/Rob--W/cors-anywhere/
But I think we need more discussion on that. Thus, targeting desktop and mobile app is good enough for now and we can disable that feature on web version and take care of it later.

Reply:
We need a proxy server to deal with the cors issue for web app. And I don't wanna have the proxy server just for this feature because someone have to pay for it. So I think this feature should be available just for desktop app users(And mobile app users one day)

Reply:
@ZeroX-DG @Rokt33r 
I fixed the feature to be only for desktop and (partly) mobile version application.
Mobile platform detection using UserAgent is not perfect, so we may need to improve it in the future.